<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrigonometryPage</class>
 <widget class="QWidget" name="TrigonometryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>

  <layout class="QVBoxLayout" name="verticalLayout">
   <!-- Title -->
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="text">
      <string>Trigonometric Functions</string>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>

   <!-- Subtitle -->
   <item>
    <widget class="QLabel" name="labelSubtitle">
     <property name="text">
      <string>All angles are in degrees</string>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
    </widget>
   </item>

   <!-- Input Section -->
   <item>
    <widget class="QWidget" name="inputWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed"/>
     </property>

     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelAngle">
        <property name="text">
         <string>Angle (°)</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QLineEdit" name="lineEditAngle">
        <property name="placeholderText">
         <string>Enter angle in degrees</string>
        </property>
       </widget>
      </item>

      <item>
       <widget class="QPushButton" name="buttonCalculate">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Results Section -->
   <item>
    <widget class="QGroupBox" name="groupBoxResults">
     <property name="title">
      <string>Results</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding"/>
     </property>

     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSin">
        <property name="text">
         <string>sin(θ)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelSinValue">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="labelCos">
        <property name="text">
         <string>cos(θ)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="labelCosValue">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>

      <item row="2" column="0">
       <widget class="QLabel" name="labelTan">
        <property name="text">
         <string>tan(θ)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="labelTanValue">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>

      <item row="3" column="0">
       <widget class="QLabel" name="labelSec">
        <property name="text">
         <string>sec(θ)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="labelSecValue">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>

      <item row="4" column="0">
       <widget class="QLabel" name="labelCsc">
        <property name="text">
         <string>csc(θ)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="labelCscValue">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>

      <item row="5" column="0">
       <widget class="QLabel" name="labelCot">
        <property name="text">
         <string>cot(θ)</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="labelCotValue">
        <property name="text">
         <string>—</string>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::TextSelectableByMouse</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>

 <resources/>
 <connections/>
</ui>
